src/variational/advi.jl

```diff
@@ -1,18 +1,14 @@
-using StatsFuns
-using DistributionsAD
-using Bijectors
-using Bijectors: TransformedDistribution
-using Random: AbstractRNG, GLOBAL_RNG
-import Bijectors: bijector
-
 """
-    bijector(model::Model; sym_to_ranges = Val(false))
+    bijector(model::Model[, sym2ranges = Val(false)])
 
 Returns a `Stacked <: Bijector` which maps from the support of the posterior to ℝᵈ with `d`
 denoting the dimensionality of the latent variables.
 """
-function bijector(model::Model; sym_to_ranges::Val{sym2ranges} = Val(false)) where {sym2ranges}
-    varinfo = Turing.VarInfo(model)
+function Bijectors.bijector(
+    model::DynamicPPL.Model,
+    ::Val{sym2ranges} = Val(false),
```

Comment:
I just noted that somehow I switched from keyword arguments to positional arguments (I guess since otherwise the name of the argument can't be omitted) - I guess we want to keep the keyword argument version?